commit d9337f555d1bb12b572651e6bd1af088f0cc6f60 (refs/changes/46/5146/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2018-11-28T18:38:46+00:00 (10 months ago)
    
    MANTA-4036 Unauthenticated requests broken by MANTA-3983
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/lib/auth.js b/lib/auth.js
index 0718bce..37205e5 100644
--- a/lib/auth.js
+++ b/lib/auth.js
@@ -782,7 +782,7 @@ function getActiveRoles(req, res, next) {
      * status, if present).
      */
     var account = req.caller.account;
-    if (account.groups.indexOf('role-operators') !== -1) {
+    if (account.groups && account.groups.indexOf('role-operators') !== -1) {
         /*
          * The req.caller object is cached and potentially shared between
          * multiple requests. We're going to alter the isOperator flag on
